cmd/hypersdk-cli/endpoint_set.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ava-labs/hypersdk/api/jsonrpc"
+)
+
+var endpointSetCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set the endpoint URL",
+	RunE: func(cmd *cobra.Command, _ []string) error {
+		endpoint, err := cmd.Flags().GetString("endpoint")
+		if err != nil {
+			return fmt.Errorf("failed to get endpoint flag: %w", err)
+		}
+
+		if endpoint == "" {
+			return errors.New("endpoint is required")
+		}
+
+		if err := updateConfig("endpoint", endpoint); err != nil {
+			return fmt.Errorf("failed to update config: %w", err)
+		}
+
+		client := jsonrpc.NewJSONRPCClient(endpoint)
+		success, err := client.Ping(context.Background())
+		pingErr := ""
+		if err != nil {
+			pingErr = err.Error()
+		}
```

Comment:
(unnecessary for this PR) - I think we should support setting the endpoint to a value that is not currently running. If we're going to support that case, should we remove the ping? This currently supports it, but will print out an error result. This is not unexpected if the user knows the endpoint is not currently running, but the UX could be better without the error message.